docker/README.md

```diff
@@ -115,7 +115,7 @@ bob$ lncli getinfo
 $ docker inspect "bob" | grep IPAddress
 
 # Connect "Alice" to the "Bob" node:
-alice$ lncli connect <bob_pubkey>@<bob_host>:10011
+alice$ lncli connect <bob_pubkey>@<bob_host>:5656
```

Comment:
With the latest set of commits, specifying the target port is no longer necessary. The `lncli` command will default to the current default p2p port if one isn't specified. As a result, you can leave off the port here entirely.